src/templates/Checkout/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Layout from 'atoms/Layout';
+import Col from 'atoms/Layout/Col';
+import { Partial, separatePartials, partialRenderer } from 'atoms/Partial';
+import CheckOutPartials from './CheckOutPartials';
+
+import styles from './checkout.module.scss';
+
+function CheckOut(props) {
+  const {
+    children,
+    className,
+  } = props;
+
+  const parts = separatePartials(children);
+  const renderPartial = partialRenderer(CheckOutPartials);
+
+  return (
+    <div className={classnames(styles['checkout'], className)}>
+      <Layout
+        className={styles.layout}
+        fullwidth
+      >
+        <Col className={styles.head}>
+          <Layout
+            mediumCols={[ 4 ]}
+            style={{ justifyContent: 'space-between' }}
```

Comment:
@patrickkim There's no real reason not to do this via a class in the CSS, is there?